binr/rabin2/rabin2.c

```diff
@@ -358,7 +358,9 @@ static int rabin_do_operation(const char *op) {
 		RBinPlugin *plg = r_bin_file_cur_plugin (cur);
 		if (!plg) break;
 		if (plg->signature) {
-			plg->signature (cur);
+			char *sign = plg->signature (cur);
```

Comment:
there shouldn't be memleak as long is just return bin->signature and when freeing the mach bin object it does the free. I can add const char * + plus casting. 1m